src/broker/Manager.cc

```diff
@@ -716,7 +753,7 @@ bool Manager::PublishLogWrite(EnumVal* stream, EnumVal* writer, string path, int
     lb.msgs[topic].add(std::move(msg));
 
     if ( lb.message_count >= log_batch_size )
-        statistics.num_logs_outgoing += lb.Flush(bstate->endpoint, log_batch_size);
+        num_logs_outgoing_metric->Inc(static_cast<double>(lb.Flush(bstate->endpoint, log_batch_size)));
```

Comment:
Maybe on two lines for clarity?

Reply:
That's just how clang-format formats it.

Reply:
I thought of having an intermediate value that's then passed to `Inc()` :-)